scripts/upload-assets.js

```diff
@@ -122,6 +125,19 @@ async function uploadAssets() {
       true
     );
 
+    await putObject(
+      fileName,
+      `${type}/${bufferClientName}/${version}/${file.includes('map') ? file : bufferFile}`,
```

Comment:
It might be safer to add a `.` prefix to `map` just in case we ever add an integration that has `map` in its name (we don't currently, but you never know!)
```suggestion
      `${type}/${bufferClientName}/${version}/${file.includes('.map') ? file : bufferFile}`,
```